src/onnx/onnx.cpp

```diff
@@ -135,16 +135,18 @@ struct onnx_parser
         add_mem_op("ConvTranspose", &onnx_parser::parse_conv_transpose);
         add_mem_op("Dropout", &onnx_parser::parse_dropout);
         add_mem_op("Elu", &onnx_parser::parse_elu);
-        add_mem_op("Equal", &onnx_parser::parse_equal);
+        add_mem_op("Equal", &onnx_parser::parse_compare_op);
```

Comment:
I think there should be another input for this function call, which is:
add_mem_op("Equal", "equal", &onnx_parser::parse_compare_op);

Reply:
Added